lib/test_alertwildfire_API.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 The Fuego Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+
+Test alertwildfire_API
+
+"""
+
+import alertwildfire_API
+import pytest
+import os
+
+def test_get_all_camera_info():
+	#get_all_camera_info()
+	loc= alertwildfire_API.get_all_camera_info()
+	assert type(loc)==type([])
+	assert type(loc[0])==type({})
+
+def test_get_individual_camera_info():
+	#get_individual_camera_info(cameraID)
+	c = alertwildfire_API.get_individual_camera_info("Axis-UpperBellNorth")
+	assert type(c)==type({})
+	assert type(c["name"])==type('')
+	c = alertwildfire_API.get_individual_camera_info("testingfailcase")
+	assert type(c)==type(None)
+
+def test_request_current_image():
+	#request_current_image(outputDir,cameraID,closestTime = None,display=False)
+	os.mkdir("tempdir")
+	imgPath = alertwildfire_API.request_current_image("./tempdir","Axis-UpperBellNorth",closestTime = None)
+	assert os.path.isfile(imgPath)
+	os.remove(imgPath)
+	os.rmdir("tempdir")
+
+
+def test_request_all_current_images():
```

Comment:
As mentioned in the other file, let's not test this expensive function and instead let's disable it.